Flaky TF FFT unit test

So I keep getting intermittent test failures on GH Actions with the 2D FFT unit test:
```
=================================== FAILURES ===================================
____________________ TestFFT.test_fft[TensorFlowBackend2D] _____________________

self = <test_tensorflow_backend_2d.TestFFT object at 0x7f093716c9a0>
backend = <class 'kymatio.scattering2d.backend.tensorflow_backend.TensorFlowBackend2D'>

    @pytest.mark.parametrize('backend', backends)
    def test_fft(self, backend):
        x = np.random.randn(2, 2)
    
        y = np.array([[x[0, 0] + x[0, 1] + x[1, 0] + x[1, 1],
                       x[0, 0] - x[0, 1] + x[1, 0] - x[1, 1]],
                      [x[0, 0] + x[0, 1] - x[1, 0] - x[1, 1],
                       x[0, 0] - x[0, 1] - x[1, 0] + x[1, 1]]])
    
        z = backend.rfft(x)
>       assert np.allclose(y, z)
E       assert False
E        +  where False = <function allclose at 0x7f098b65b550>(array([[ 2.95330393e+00, -4.98718912e-01],\n       [-1.51491472e-03,  7.56337531e-01]]), <tf.Tensor: shape=(2, 2), dtype=complex64, numpy=\narray([[ 2.9533038e+00+0.j, -4.9871886e-01+0.j],\n       [-1.5148520e-03+0.j,  7.5633746e-01+0.j]], dtype=complex64)>)
E        +    where <function allclose at 0x7f098b65b550> = np.allclose

tests/scattering2d/test_tensorflow_backend_2d.py:141: AssertionError
```
It doesn't reproduce locally. Likely this has to do with the cheap FFT implementation that they use (although it's surprising that it fails for such small arrays). The thing to do might be to relax the precision a bit.

--- a/tests/scattering2d/test_tensorflow_backend_2d.py
+++ b/tests/scattering2d/test_tensorflow_backend_2d.py
@@ -130,6 +130,10 @@ class TestCDGMM:
 class TestFFT:
     @pytest.mark.parametrize('backend', backends)
     def test_fft(self, backend):
+        # NOTE: The CPU implementation used by TF is not very accurate, so we
+        # have to relax the tolerances a little here (default rtol is 1e-5).
+        rtol = 1e-4
+
         x = np.random.randn(2, 2)
 
         y = np.array([[x[0, 0] + x[0, 1] + x[1, 0] + x[1, 1],
@@ -138,14 +142,14 @@ class TestFFT:
                        x[0, 0] - x[0, 1] - x[1, 0] + x[1, 1]]])
 
         z = backend.rfft(x)
-        assert np.allclose(y, z)
+        assert np.allclose(y, z, rtol=rtol)
         
         z_1 = backend.irfft(z)
         assert not np.iscomplexobj(z_1)
-        assert np.allclose(x, z_1)
+        assert np.allclose(x, z_1, rtol=rtol)
 
         z_2 = backend.ifft(z)
-        assert np.allclose(x, z_2)
+        assert np.allclose(x, z_2, rtol=rtol)
 
     @pytest.mark.parametrize('backend', backends)
     def test_fft_type(self, backend):

